Scoop.js

```diff
@@ -1008,18 +1008,20 @@ export class Scoop {
         }
       }
 
-      // Merge parsed metadata into a single JSON string and clean it before saving it
-      const metadataAsJSON = JSON
-        .stringify(metadataParsed, null, 2)
-        .replaceAll(this.captureTmpFolderPath, '')
+      if (metadataParsed.length) {
```

Comment:
Instead of nesting, it could be justified to throw an exception if metadataParsed.length < 1